gslab_scons/provenance.py

```diff
@@ -100,11 +101,13 @@ def scan(start_path, include_details, include_checksum, detail_limit, file_detai
     return total_size, num_files, last_mtime, file_details, dirs
 
 
-def write_heading(start_path, provenance_path):
+def write_heading(start_path, provenance_path, github_release = None):
```

Comment:
No need to set the default value for github_release in the sub-function since it's already set globally in make_provenance.

Reply:
I prefer having default option for helper functions

Reply:
Makes sense. Can you implement everywhere then.